packages/dataviews/src/view-table.js

```diff
@@ -366,14 +416,19 @@ function ViewTable( {
 								) }
 							</tr>
 						) ) }
-					</tbody>
-				</table>
-			) }
-			{ ! hasData && (
-				<div className="dataviews-no-results">
-					<p>{ __( 'No results' ) }</p>
-				</div>
-			) }
+				</tbody>
+			</table>
+			<div
+				className={ classNames( 'dataviews-table-status', {
+					'dataviews-loading': isLoading,
+					'dataviews-no-results': ! hasData && ! isLoading,
+				} ) }
```

Comment:
I don't see the classes being used for anything. Perhaps we could remove them for now?

```suggestion
```

Reply:
Ah, nice. Searched for them, but didn't see them. I must have done the wrong search.